main.py

```diff
@@ -0,0 +1,120 @@
+from fastapi import FastAPI, Header, Request
+from pydantic import BaseModel
+import re
+import pymysql.cursors
+
+app = FastAPI()
+
+
+class Login(BaseModel):
+    user_id: str
+    user_pw: str
+
+
+@app.get("/")
+async def root():
+    return {"message": "Hello World"}
+
+
+"""
+API handler that functions the followings;
+1. validate the user id in accordance with regulations
+2. validate the user password in accordance with regulations
+3. checks if the given credential equates with the data in DB
+4. handles the HTTP errors
+"""
+
+
+@app.post("/api/v1/login")
+async def usr_login(usr_login: Login, request: Request):
+    headers = request.headers
+
+    success_format = {
+        "ifLogin": True,
+        "roomCode": "rushhour",
+        "status": "200",
+        "statusMessage": "Success",
+    }
+
+    fail_format = {
+        "ifLogin": False,
+        "status": "500",
+        "statusMessage": "Invalid Format",
+    }
+
+    # 1. ID : a-zA-Z, length: less than 15 characters
+    def userIdCheck(user_id) -> bool:
+        if user_id is None:  # null handler
+            return False
+
+        id_check = re.findall("[a-zA-Z]", user_id)
+        if len(user_id) == len(id_check) and len(user_id) <= 15:
+            return True
+        else:
+            return False
+
+    # 2. Password : a-zA-Z0-9 + one special character length, more than 8, less than 20 characters
```

Comment:
주석을 가독성 높이기 위해 라인을 나눠서 표시부탁드립니다. 
제 생각에는 length, (엔터)후 more부터 다음라인으로 되는게 좋다 생각됩니다.

ex>
```python
# 2. Password : a-zA-Z0-9 + one special character length, 
# more than 8, less than 20 characters.
```

Reply:
All methods were annotated per your suggestion. Thank you!
The commit below may no longer be available since the structure has been overall refactored.
`RushHour_/pull/8/commits/9808fd1866ab35b2de1b57e5e36f5699a76ce49b#diff-b10564ab7d2c520cdd0243874879fb0a782862c3c902ab535faabe57d5a505e1R30-R36`